### ISSUE TITLE
New-StoredCredential should also accept securestring as password

### ISSUE THREAD

**Issue by majkinetor** [davotronic5000/PowerShell_Credential_Manager#9]
I currently do this boilerplate:
  
    try { $cred = Get-StoredCredential $url } catch {}
    if ($cred -eq $null) {
        $user = Read-Host -Prompt "Username ($Env:USERDOMAIN\$Env:USERNAME)"
        $pass = Read-Host -Prompt "Password" -AsSecureString
        $cred = New-Object System.Net.NetworkCredential('', $pass)
        New-StoredCredential -Target $url $cred.password
        Remove-Variable cred
    }

It would also be easier if `Get-StoredCredential` would use non terminating error so we could use `-ea o` instead of `try/catch` block: 
 
    $c = Get-StoredCredential $url -ea 0
    if (!$c) {...}

**Comment by davotronic5000** [comment]
Thanks for the feed back.  I have fixed these things in the development branch.

I have removed all the terminating errors, so all the cmdlets will now correctly respect the Erroracction preference set,  for this specific case where a requested credential is not found, I have actually removed the error completely the command will complete successfully but return no data which would be the expected behavior in powershell.

I have also added an additional parameter to the New-StoredCredential command which will accept a secure string as the password.

This is all currently sitting in the development branch which you are welcome to download and compile if you are comfortable with it.  I will roll these changes in to the next release but have some more work to do before I can do that.
